homeassistant/components/config/device_registry.py

```diff
@@ -47,14 +47,18 @@ def websocket_list_devices(
         f'"success":true,"result": ['
     ).encode()
     # Concatenate cached entity registry item JSON serializations
-    msg_json = (
-        msg_json_prefix
-        + b",".join(
-            entry.json_repr
-            for entry in registry.devices.values()
-            if entry.json_repr is not None
+    msg_json = b"".join(
```

Comment:
`b"".join` so the fast path can be used https://github.com/python/cpython/blob/f383ca1a6fa1a2a83c8c1a0e56cf997c77fa2893/Objects/stringlib/join.h#L123